UI/src/view/HeightLimitLabel.js

```diff
@@ -0,0 +1,55 @@
+(function(root) {
+
+  root.HeightLimitLabel = function(){
+    AssetLabel.call(this, this.MIN_DISTANCE, this.populatedPoints);
+    var me = this;
+    var IMAGE_HEIGHT = 23;
+    this.MIN_DISTANCE = 5;
+    this.populatedPoints = [];
+
+    me.renderFeaturesByPointAssets = function(pointAssets, zoomLevel){
+      me.clearPoints();
+      return me.renderGroupedFeatures(pointAssets, zoomLevel, function(asset){
+        return me.getCoordinateForGrouping(asset);
```

Comment:
Considering the suggestion given on the AssetLabel.js file in here we can just use the  `return me.getCoordinate(asset);`